source/postprocess/composition_velocity_statistics.cc

```diff
@@ -153,7 +153,7 @@ namespace aspect
       {
         prm.enter_subsection("Composition velocity statistics");
         {
-          prm.declare_entry("Names of slab compositional fields", "",
+          prm.declare_entry("Names of selected compositional fields", "",
```

Comment:
The text of this parameter needs to be updated as well, i.e. 'the subducting plate' should be changed.